src/config.js

```diff
@@ -0,0 +1,62 @@
+/* @flow */
+import path from 'path';
+
+import requireUncached from 'require-uncached';
+
+import {createLogger} from './util/logger';
+import {UsageError} from './errors';
+
+const log = createLogger(__filename);
+
+type ApplyConfigToArgvParams = {|
+  argv: Object,
+  configObject: Object,
+  defaultValues?: Object,
```

Comment:
This should not be optional. I only see a few cases in your tests where it's not passed but you can update those calls to pass in the object. This function needs to operate on `defaultValues` for it to do its job.